### PR TITLE
add select on login page and Services.js file

### PR DESCRIPTION
**Pull request by aberonja**
Created Select.js component and appropriate style in _login.scss file
Removed export from GuestLogin.js and added Select component
Created Services.js file for functions
Added color change function for select component in Services.js file
Fixed responsive issues on start screen

### CODE REVIEW

File: web/src/main/frontend/src/base/Services.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const selectChangeColor = () => {
+  document.getElementById('test-type').style.color = 'white';
```
Comment (iscekic) [inline web/src/main/frontend/src/base/Services.js]: `getElementById`? :no_good_man:

File: web/src/main/frontend/src/pages/GuestLogin.js [hunk 2]
```diff
@@ -36,12 +37,18 @@ export class GuestLogin extends React.Component {
                 placeholder="Please enter your email"
               />
             </div>
+            <div className="col-12 col-md-3 text-left">
+              <label htmlFor="testType">Test type</label>
```
Comment (iscekic) [inline web/src/main/frontend/src/pages/GuestLogin.js]: IDs should use hyphens.

File: web/src/main/frontend/src/base/Services.js (hunk 1 above)
Comment (aberonja) [reply web/src/main/frontend/src/base/Services.js]: What alternative do you suggest? :smile:

Comment (iscekic) [reply web/src/main/frontend/src/base/Services.js]: Local component state.

File: web/src/main/frontend/src/components/Select.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+class Select extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      change: false,
+      changeFunc: function handleChange() {
```
Comment (iscekic) [inline web/src/main/frontend/src/components/Select.js]: Functions shouldn't be part of `state`. Use a method:
```js
class ... extends ... {
  changeColor = () => {
    this.setState({
      color: 'new color'
    });
  }
}
```

File: web/src/main/frontend/src/components/Select.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+class Select extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```
Comment (iscekic) [inline web/src/main/frontend/src/components/Select.js]: You can avoid writing a constructor by using this syntax:
```js
class ... extends ... {
  state = {
    color: 'initial color'
  }
}
```

File: web/src/main/frontend/src/components/Select.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      change: false,
+      changeFunc: function handleChange() {
+        this.setState({change: true});
+      }
+    };
+    this.state.changeFunc = this.state.changeFunc.bind(this);
+  }
+
+  render() {
+    const selectStyle = {
```
Comment (iscekic) [inline web/src/main/frontend/src/components/Select.js]: Remove this variable.

File: web/src/main/frontend/src/components/Select.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+      color: 'white'
+    };
+    return (
+      <div className="col-12 col-md-9 text-left">
+        <select
+          id="test-type"
+          style={this.state.change ? selectStyle : null}
```
Comment (iscekic) [inline web/src/main/frontend/src/components/Select.js]: This line should be:
```js
style={{
  color: this.state.color
}}
```

File: web/src/main/frontend/src/components/Select.js [hunk 7]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+class Select extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      change: false,
+      changeFunc: function handleChange() {
+        this.setState({change: true});
```
Comment (iscekic) [inline web/src/main/frontend/src/components/Select.js]: You should never `setState` in the constructor.

File: web/src/main/frontend/src/components/Select.js [hunk 8]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+class Select extends React.Component {
+  state = {
+    color: null
```
Comment (iscekic) [inline web/src/main/frontend/src/components/Select.js]: `null` is not a valid initial color. :smile: Or does React handle that case by not inserting CSS?

Comment (aberonja) [reply web/src/main/frontend/src/components/Select.js]: React handles it. There is no style on this element before change.

**Pull request merged**